### PR TITLE
Test Ansible Girder module with Molecule

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
I'm going to clean this up a little more, but note that the  Ansible Girder module was broken by #3045. @kotfic would you like to take a look?

**Comment by kotfic** [comment]
Sure, did you mean take a look at this PR or at the module post #3045?

**Comment by brianhelba** [comment]
@kotfic This PR fixes the infrastructure to run the existing tests for the Ansible module. There's not much I need you to review in the diffs of this PR (yet). The unsolved issue is that since these tests had not been running for several months, it turns out that #3045 caused the Ansible module to break, and we didn't know about it.

The now-failing tests start at about https://github.com/girder/girder/blob/molecule-module/devops/ansible-role-girder/molecule/ansible-module/test_setting.yml#L73 , the underlying problem seems to be in the code at https://github.com/girder/girder/blob/molecule-module/devops/ansible-role-girder/library/girder.py#L1762 .

File: devops/ansible-role-girder/library/girder.py [hunk 1]
```diff
@@ -1764,20 +1758,8 @@ def setting(self, key, value=None):
...
-                try:
-                    self.delete('system/setting', parameters={'key': key})
-                    self.changed = True
-
-                    ret['previous_value'] = existing_value
-                    ret['current_value'] = default
-                except requests.HTTPError as e:
-                    self.fail(e.response.json()['message'])
+        else:
+            self.fail('%s state isn\'t supported by Girder setting module.' % self.module.params['state'])
```
Comment (brianhelba) [inline devops/ansible-role-girder/library/girder.py]: @danlamanna Line too long.

**Pull request merged**